openverse_catalog/dags/providers/provider_api_scripts/provider_data_ingester.py

```diff
@@ -160,7 +166,7 @@ def _init_media_stores(self) -> dict[str, MediaStore]:
 
         for media_type, provider in self.providers.items():
             StoreClass = get_media_store_class(media_type)
-            media_stores[media_type] = StoreClass(provider)
+            media_stores[media_type] = StoreClass(provider, tsv_suffix=str(day_shift))
```

Comment:
Oh, it's this line 😅 This is converting `None` and `0` to `'None'` and `'0'` which don't pass the filter! So they're getting appended to the filename anyway.